ebu_tt_live/bindings/__init__.py

```diff
@@ -835,12 +1092,16 @@ def __check_bds(cls, bds):
             )
 
     def toDOM(self, bds=None, parent=None, element_name=None):
+        bds = self.__check_bds(bds)
         xml_dom = super(d_tt_type, self).toDOM(
-            bds=self.__check_bds(bds),
+            bds=bds,
             parent=parent,
             element_name=element_name
         )
-        xml_dom.documentElement.tagName = 'tt:tt'
+        if bds.defaultNamespace() != Namespace:
+            xml_dom.documentElement.tagName = 'tt:tt'
+        else:
+            xml_dom.documentElement.tagName = 'tt'
```

Comment:
Well in fairness this is an ugly hack that I did not have enough time to execute as well as I should have. The problem is the following: This is not meant to be done at all ever but since EBU keeps redefining the same tt element in the ttml namespace for EBU-TT-D I needed to rename it to ttd to stop PyXB from blowing up.